docker/bigtop-puppet/build.sh

```diff
@@ -63,5 +64,10 @@ case "${OS}-${VERSION}" in
     ;;
 esac
 
-docker build -t bigtop/puppet:${PREFIX}-${OS}-${VERSION}${ARCH} .
+if [ $OS == "openeuler/openeuler" ];then
+  docker build -t ${PREFIX}-${OS}-${VERSION}${ARCH} .
```

Comment:
The build command should be : `./build.sh trunk-openeuler-20.3`,
not be `./build.sh trunk-openeuler/openeuler-22.03`

Reply:
@guyuqi  thx, it will be modified.

Reply:
@guyuqi The build command was modifed, the commit:
https://github.com/apache/bigtop/pull/1051/commits/a46e50e432b830907deca32bfdbaf63dabbb1ee7

Now, the build command is :  **./build.sh trunk-openeuler-20.3**

Reply:
> Now, the build command is : **./build.sh trunk-openeuler-20.3**

you mean "22.03", right?

Reply:
@docularxu yes